tests/test_trackcompare.py

```diff
@@ -100,6 +100,23 @@ def test_extract_no_spatial_overlap_dfs0(modelresult, observation_df):
     assert len(cc.all_df) == 0
 
 
+# def test_extract_from_model_result_with_nan(sw_Hm0_df, Hm0_EPL):
```

Comment:
Why in `test_trackcompare`?

Reply:
That was probably a mistake - my primary concern was to move it out of ModelResult :-)